code/modules/jobs/job_types/prisoner.dm

```diff
@@ -13,6 +14,10 @@
 	plasmaman_outfit = /datum/outfit/plasmaman/prisoner
 
 	display_order = JOB_DISPLAY_ORDER_PRISONER
+	department_for_prefs = /datum/job_department/security
+	departments_list = list(
+		/datum/job_department/service,
```

Comment:
HOLY SHIT IT'S BACK BABY IT'S FUCKING BACK HOLY SHIT

Reply:
Wait I didn't even mean to have both of these

Reply:
NOOOOOOOOOOOOOOOO